src/core/prometheus.cc

```diff
@@ -481,6 +481,24 @@ metric_family_range get_range(const metrics_families_per_shard& mf, const sstrin
 
 }
 
+void write_counter(std::stringstream& s, const config& ctx, const sstring& name, const mi::metric_value& value, std::map<sstring, sstring> labels) {
```

Comment:
If this function was called in this bit of the refactor, the next commit would be easier to review.